docs/tutorials/excel-power-automate-returns.md

```diff
@@ -0,0 +1,183 @@
+
+    ```typescript
+    // Look for the first row where today's date is between the row's start and end dates.
+    let currentDate = new Date();
+    for (let row = 0; row < tableValues.length; row++) {
+        let startDate = convertDate(tableValues[row][2] as number);
+        let endDate = convertDate(tableValues[row][3] as number);
+        if (startDate <= currentDate && endDate >= currentDate) {
+            // Return the first matching email address.
+            return tableValues[row][1].toString();
+        }
+    }
+    ```
+
+1. The final script should look like this:
+
+    ```typescript
+    function main(workbook: ExcelScript.Workbook) : string {
+        // Get the H1 worksheet.
+        let worksheet = workbook.getWorksheet("H1");
+
+        // Get the first (and only) table in the worksheet.
+        let table = worksheet.getTables()[0];
+    
+        // Get the data from the table.
+        let tableValues = table.getRangeBetweenHeaderAndTotal().getValues();
+    
+        // Look for the first row where today's date is between the row's start and end dates.
+        let currentDate = new Date();
+        for (let row = 0; row < tableValues.length; row++) {
+            let startDate = convertDate(tableValues[row][2] as number);
+            let endDate = convertDate(tableValues[row][3] as number);
+            if (startDate <= currentDate && endDate >= currentDate) {
+                // Return the first matching email address.
+                return tableValues[row][1].toString();
+            }
+        }
+    }
+
+    // Convert the Excel date to a JavaScript Date object.
+    function convertDate(excelDateValue: number) {
+        let javaScriptDate = new Date(Math.round((excelDateValue - 25569) * 86400 * 1000));
+        return javaScriptDate;
+    }
+    ```
+
+## Create an automated workflow with Power Automate
+
+1. Sign in to the [Power Automate site](https://flow.microsoft.com).
+
+1. In the menu that's displayed on the left side of the screen, press **Create**. This brings you to list of ways to create new workflows.
+
+    ![The Create button in Power Automate](../images/power-automate-tutorial-1.png)
+
+1. Under the **Start from blank** section, select **Scheduled cloud flow**.
+
+    ![The Scheduled cloud flow button in Power Automate](../images/power-automate-return-tutorial-2.png)
+
+1. Now we need to set the schedule for this flow. Our spreadsheet has a new on-call assignment starting every Monday in the first half of 2021. Let's set the flow to run first thing Monday mornings. Use the following options to configure the flow to run on Monday each week:
+
+    - **Flow name**: Notify On-Call Person
+    - **Starting**: 1/4/21 at 1:00am
+    - **Repeat every**: 1 Week
+    - **On these days**: M
+
+    ![A window with the specified options for the scheduled flow](../images/power-automate-return-tutorial-3.png)
```

Comment:
Looks like the top of the image has a red dot that shouldn't be there...?

Reply:
Good catch. Cropped that out of the image.